src/einsteinpy/symbolic/ricci.py

```diff
@@ -96,3 +154,74 @@ def symbols(self):
 
         """
         return self.syms
+
+
+class RicciScalar:
+    """
+    Class for defining Ricci Scalar
+    """
+
+    def __init__(self, metric_tensor, ricci_tensor):
+        """
+        Constructor and Initializer
+
+        Parameters
+        ----------
+        metric_tensor : ~einsteinpy.symbolic.metric.MetricTensor
+            Metric Tensor from which Ricci Scalar to be calculated
+        ricci_tensor : ~einsteinpy.symbolic.metric.RicciTensor
+            Ricci Tensor from which Ricci Scalar to be calculated
+
+        Raises
+        ------
+        TypeError
+            Raised when metric_tensor is not a MetricTensor or/and ricci_tensor is not a RicciTensor
+
+        """
+        if isinstance(metric_tensor, MetricTensor) and isinstance(
+            ricci_tensor, RicciTensor
+        ):
+            self.ricci_scalar = self.calculate_ricci_scalar(metric_tensor, ricci_tensor)
+
+        else:
+            raise TypeError(
+                "metric_tensor should be a MetricTensor class and ricci_tensor should be a RicciTensor"
+            )
+
+    def calculate_ricci_scalar(self, metric_tensor, ricci_tensor):
```

Comment:
remove this function, we are going with classmethods

Reply:
First do this much as suggested by the review, then we would make the classmethods